Source/WebCore/rendering/RenderThemeCocoa.h

```diff
@@ -25,7 +25,11 @@
 
 #pragma once
 
+#include "FloatRoundedRect.h"
+#include "GraphicsContextCG.h"
+#include "Icon.h"
 #include "RenderTheme.h"
+#include <CoreGraphics/CoreGraphics.h>
```

Comment:
Are any of the imports actually needed in this header? We should import in the implementation files when possible.